### PR TITLE
Add Elvish Explorers Vault

### PR DESCRIPTION
**Pull request by kippig**
This vault adds an elvish couple exploring the dungeon with their
mounts and their pet dog. Sometimes they have starved to death
since they can't find enough rat to skewer. In crypt they have
succumbed and been raised. They have a nice campfire and when in
depths they are brave enough to leave it (non-patrolling).

### CODE REVIEW

**Comment by kippig** [comment]
Talked to gammafunk offline. Redoing the map to make it interesting.

**Review by gammafunk — COMMENTED** [review]
You mentioned that you might rework the layout of this vault to better communicate the theme, so I kept my review general. Look forward to the final version!

File: crawl-ref/source/dat/des/variable/mini_monsters.des [hunk 1]
```diff
@@ -2232,6 +2232,52 @@ MAP
...
+MONS: fungus / nothing
+: elseif you.in_branch("Crypt") then
+MONS: deep elf master archer zombie / deep elf master archer skeleton, \
+      deep elf blademaster zombie / deep elf blademaster skeleton
+MONS: dire elephant zombie / dire elephant skeleton
+MONS: raiju skeleton / raiju zombie
+MONS: nothing
+: else
+MONS: deep elf master archer, deep elf blademaster
+MONS: dire elephant
```
[the remark this answers is not in the document]
Comment (kippig) [reply crawl-ref/source/dat/des/variable/mini_monsters.des:2263]: I like this idea.

**Comment by kippig** [comment]
Ready for map review

**Pull request closed without merging**